src/Html/Rules/NoTextEmptyString.elm

```diff
@@ -0,0 +1,106 @@
+module Html.Rules.NoTextEmptyString exposing (error, rule)
+
+{-| @doc rule
+-}
```

Comment:
```suggestion
{-|
@docs rule
-}
```

The `@docs` can't be on the first line and they need to be at the beginning of the line. Also it's `@docs`. I recommend https://www.npmjs.com/package/elm-doc-preview for visualizing the docs before something gets published. I found it really useful.